### PR TITLE
Add support for Species test data generation.

### PR DESCRIPTION
**Pull request by meyumeapps**

### CODE REVIEW

File: service/species-service.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    log.info(`${SERVICE_NAME}::createSpecies:url:${url}`)
+    return axios.post(url, payload)
+  } catch (error) {
+    log.error(error)
+  }
+}
+
+async function deleteSpecies (productNo) {
+  try {
+    let url = ADMIN_BASE_API_URL + `/species/productNo/${productNo}`
```
Comment (matt-gregory) [inline service/species-service.js]: productNo?

File: service/species-service.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+const axios = require('axios')
+const log = global.log
+const ADMIN_BASE_API_URL = `${process.env.ADMIN_BASE_API_URL}`
+const SERVICE_NAME = 'species-service'
+
+async function createSpecies (payload) {
+  try {
+    let url = ADMIN_BASE_API_URL + '/species/product'
```
Comment (matt-gregory) [inline service/species-service.js]: /species/product?

File: service/species-service.js (hunk 1 above)
Comment (meyumeapps) [reply service/species-service.js]: The species is associated with a product. We are going to allow a user to register a new product and at the same time a species. There are no checks to on the species name apparently. Could be anything!

Hence, we can only get to the species using the referenced product number. Using the name wouldn't be unique.

File: service/species-service.js (hunk 2 above)
Comment (meyumeapps) [reply service/species-service.js]: Yep, the species is associated with a product. I've gone this route for now as I suspect we will eventually be able to create a species disconnected from a product.

**Pull request merged**